### ISSUE TITLE
New [Run/Debug Code] button unable to run code from custom location

### ISSUE THREAD

**Issue by thefatwhale** [microsoft/vscode-cpptools#9082]
Bug type: Debugger

**Describe the bug**
- OS and Version: Microsoft Windows 7 x64 SP1
- VS Code Version: 1.63.2
- C/C++ Extension Version: 1.9.7
- Other extensions you installed (and if the issue persists after disabling them):
- A clear and concise description of what the bug is.

**To Reproduce**
*Please include a code sample and `launch.json` configuration.*
1. open a C source code file
2. click button
3. 
![image](https://user-images.githubusercontent.com/50515732/159923105-9b6fc08a-3b2d-4264-be73-ba76c28a8ff3.png)
**in my tasks.json's setting executable file compile to `./bin/`**
**press <kbd>f5</kbd> or <kbd>ctrl</kbd> <kbd>f5</kbd> is ok**
**this is my launch.json**
```json
{
	"version": "0.2.0",
	"configurations": [
		{
			"name": "gcc.exe - 生成和调试活动文件",
			"type": "cppdbg",
			"request": "launch",
			"program": "${workspaceFolder}\\bin\\${fileBasenameNoExtension}.exe",
			"args": [],
			"stopAtEntry": false,
			"cwd": "${workspaceFolder}",
			"environment": [],
			"externalConsole": false,
			"MIMode": "gdb",
			"miDebuggerPath": "D:\\SoftwareInstallation\\Windows7\\ProgramFiles\\MinGW-w64\\win32\\bin\\gdb.exe",
			"setupCommands": [
				{
					"description": "为 gdb 启用整齐打印",
					"text": "-enable-pretty-printing",
					"ignoreFailures": true
				}
			],
			"preLaunchTask": "C/C++: gcc.exe 生成活动文件"
		}
	]
}
```

**Additional context**
*If applicable, please include logging by adding "logging": { "engineLogging": true, "trace": true, "traceResponse": true } in your `launch.json`*
Add any other context about the problem here including log or error messages in your Debug Console or Output windows.

**Comment by michelleangela** [comment]
@thefatwhale 
When you click on the "run/debug" button, does it display a list of which launch or task to use before launching the program?

**Comment by thefatwhale** [comment]
No, Nothing prompt me select something.

**Comment by sean-mcmanus** [comment]
This should be fixed with 1.10.2 (pre-release).
